examples/user_guide/09-Indexing_and_Selecting_Data.ipynb

```diff
@@ -486,14 +482,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "hv.output(backend='bokeh')\n",
+    "\n",
     "xs = np.arange(10)\n",
     "extents = (0, 0, 2, 10)\n",
     "curve = hv.HoloMap({(i) : hv.Curve(zip(xs, np.sin(xs)*i))\n",
     "                    for i in np.linspace(0.5, 1.5, 3)},\n",
     "                   kdims='Observation')\n",
     "all_samples = curve.collapse().to.points()\n",
     "sampled = curve.sample([0, 2, 4, 6, 8])\n",
-    "sampling = all_samples * sampled.to.points(extents=extents).options(color='r')\n",
+    "sampling = all_samples * sampled.to.points(extents=extents).opts(color='red')\n",
```

Comment:
`sampled.to.points(extents=extents).opts(color='red')` needs a handle before being composed.